tools/image-builder/README.md

```diff
@@ -0,0 +1,53 @@
+# Image Builder
+
+This directory contains the source for a docker container that will be used for
+building Bond C++ which will contain binaries of a number of Boost versions for
```

Comment:
Minor: "Bond C++. It contains binaries..."

Reply:
Thanks.